crates/block-producer/src/block_builder/prover/mod.rs

```diff
@@ -72,6 +72,7 @@ impl BlockProver {
             nullifier_root,
             note_root,
             tx_hash,
+            Digest::default(),
```

Comment:
If we set it to the default digest, we won't be able to execute any transactions against this block. Instead, we should set `kernel_root` to `TransactionKernel::kernel_root()`.

Reply:
Yeah, I set it to that just a minute before.